frontend/src/components/blocks/Blocks.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PaginationSpinner from "../utils/PaginationSpinner";
+
+import BlocksList from "./BlocksList";
+import BlocksApi, * as B from "../../libraries/explorer-wamp/blocks";
+
+import FlipMove from "../utils/FlipMove";
```

Comment:
Let's use the following grouping for the imports:

1. standard JS/node.js library imports (not applicable to frontend)
2. generic JS libraries (e.g. `bn.js`, `moment`)
3. framework imports (`import Head from "next/head";`)
4. framework core libraries (`import React from "react"`)
5. external framework libraries (`import { Row, Col } from "react-bootstrap";`)
6. common internal modules (`import BlocksApi from "../../libraries/explorer-wamp/blocks";`)
7. local internal modules sorted from higher hierarchy to local scope (`import FlipMove from "../utils/FlipMove";` > `import BlocksList from "./BlocksList";`)

Also, we should maintain alphabetical order by a package/module name (the thing after `from` keyword) inside the groups.

Please, update the files touched in this PR and apply the rule once you touch files in the future.